core/src/main/scala/com/colisweb/jrubysnesshours/core/Core.scala

```
@@ -1,11 +1,103 @@
 package com.colisweb.jrubysnesshours.core
 
-import java.time.{DayOfWeek, LocalTime}
+import java.time._
 
 object Core {
 
   case class Interval(startTime: LocalTime, endTime: LocalTime)
```

à défaut de pouvoir utiliser Refined (pas forcément pratique depuis Ruby), je propose de rajouter dans le corps de la classe un `assert(start < end)` pour "fail fast".

---

J'hésite, on devrait faire ce genre de fail / check ici ou alors plutot dans la partie communiquant avec JRuby ?

---

La classe Interval de Threeten extra s'assure de ça : https://www.threeten.org/threeten-extra/apidocs/org.threeten.extra/org/threeten/extra/Interval.html

---

Je l'ai fait dans ma PR: https://github.com/Colisweb/JRubysnessHours/pull/5/files#diff-4bd90dcefda7c996d665c7b1c807af8cR56